apps/nextjs/src/app/api/health/live/route.ts

```diff
@@ -0,0 +1,93 @@
+import { performance } from "perf_hooks";
+
+import { db } from "@homarr/db";
+import { logger } from "@homarr/log";
+import { handshakeAsync } from "@homarr/redis";
+
+export async function GET() {
+  const timeBeforeHealthCheck = performance.now();
+  const response = await executeAndAggregateAllHealthChecksAsync();
+  logger.info(`Completed healthcheck after ${performance.now() - timeBeforeHealthCheck}ms`);
+
+  if (response.status === "healthy") {
+    return new Response(JSON.stringify(response), {
+      status: 200,
+    });
+  }
+
+  return new Response(JSON.stringify(response), {
+    status: 500,
+  });
+}
+
+const executeAndAggregateAllHealthChecksAsync = async (): Promise<{
+  healthChecks: Record<string, object>;
+  status: "healthy" | "unhealthy";
+}> => {
+  const healthChecks = [
+    executeHealthCheckSafelyAsync("database", async () => {
+      const before = performance.now();
+      // sqlite driver does not support raw query execution. this is for a heartbeat check only - it doesn't matter if data is returned or not
+      await db.query.serverSettings.findFirst();
+      const after = performance.now();
+      return {
+        latency: after - before,
+      };
+    }),
+    executeHealthCheckSafelyAsync("redis", async () => {
+      const before = performance.now();
+      await handshakeAsync();
+      const after = performance.now();
```

Comment:
You could actually simplify this by only running the handshakeAsync in here and put the before and after with latency calculation into the `executeHealthCheckSafelyAsync` method